allure-report-face/src/components/attachment/AttachmentView.hbs

```diff
@@ -38,6 +38,13 @@
     <div class="pane__section">
         <object class="attachment__embed" type="image/svg+xml" data="{{sourceUrl}}">Your browser does not support SVG</object>
     </div>
+{{else if (eq type "video")}}
+    <div class="pane__section">
```

Comment:
Yes, that was a typo. We should rename all `image*` classes to `media*`.

Also you can check it: if you did it right, the result in browser will look fine.